engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -237,4 +238,8 @@ public void setzFar(float zFar) {
     public boolean isReflected() {
         return reflected;
     }
+
+    public boolean hasInSight(AABB aabb) {
```

Comment:
Because of its frustrum-dependency, this visibility testing I felt was appropriate moved into the Camera class rather than the WorldRenderer.